node/duty_execution.go

```diff
@@ -20,12 +21,7 @@ import (
 )
 
 // waitForSignatureCollection waits for inbound signatures, collects them or times out if not.
-func (n *ssvNode) waitForSignatureCollection(
-	logger *zap.Logger,
-	identifier []byte,
-	sigRoot []byte,
-	signaturesCount int,
-) (map[uint64][]byte, error) {
+func (n *ssvNode) waitForSignatureCollection(logger *zap.Logger, identifier []byte, sigRoot []byte, signaturesCount int, committiee map[uint64]*proto.Node, ) (map[uint64][]byte, error) {
```

Comment:
@nivBlox single line params is forced by linting?

Reply:
no, lint have no problem with that